internal/shared/api/handlers/handler/swagger.go

```diff
@@ -0,0 +1,19 @@
+package handler
+
+import (
+	"github.com/babylonlabs-io/staking-api-service/docs"
+	"github.com/swaggo/swag"
+	"net/http"
+)
+
+func SwaggerDoc(w http.ResponseWriter, req *http.Request) {
```

Comment:
I though that it's better to place it to some package instead of inlining in routes.go + because it's handler not related to API handlers there is no need to declare API specification and receive/return API request/response